docs/source/use_cases/discover_public_data/index.rst

```diff
@@ -1,14 +1,19 @@
-Discover Public Data
+Discover Published Data
 ========================
+Here we list the growing collection of genomic data resident in Google Cloud Platform and shared with the community.  Some datasets are open-access, such as 1,000 Genomes, whereas other datasets are controlled-access.  Each dataset page describes the source of the data and how to *access the data* or *apply for access* as applicable for the dataset.
```

Comment:
s/Here we list the/There is a/

Italicize "open-access" and "controlled-access".
Remove the italics from "access the data"